bin/build-metadata.js

```diff
@@ -214,6 +214,18 @@ function removeAllNumberKeys( obj ) {
 	return _.omitBy( obj, function( val, key ) { return /^\d+$/.test( key ); } );
 }
 
+function removeRegionCodeAndCountryDialCodeIfSameWithCountryDialCode( countryData ) {
+	for ( var key in countryData ) {
+		if ( countryData.hasOwnProperty( key ) ) {
+			if ( ! countryData[ key ].regionCode && ( countryData[ key ].countryDialCode + ( countryData[ key ].regionCode || '' ) ) === countryData[ key ].dialCode ) {
```

Comment:
I must admit that it was hard to grasp this change for me - especially this condition (the `countryData[ key ].regionCode || ''` bit is not needed, because at that point it has to be non-empty).

How about a different approach, though: don't remove `regionCode` and `countryDialCode` :) Instead, make sure they have sane values (if `countryDialCode` is empty, set it to `dialCode`; for `regionCode` - if it's empty or doesn't exits, make it empty). Then you wouldn't need `removeRegionCodeAndCountryDialCodeIfSameWithCountryDialCode` (no matter how awesome that function's name is :rofl:) and `toIcannFormat` could be much simplified:
```
export function toIcannFormat( inputNumber, country ) {
	const { nationalNumber } = processNumber( inputNumber, country ),
		{ countryDialCode, regionCode } = country;
	return '+' + countryDialCode + '.' + regionCode + nationalNumber;
}
```